tools/shell/test/test_helper.py

```diff
@@ -0,0 +1,45 @@
+import os
+from pathlib import Path
+import sys
+from enum import Enum
+
+KUZU_ROOT = Path(__file__).parent.parent.parent.parent
+if sys.platform == "win32":
+    # \ in paths is not supported by kuzu's parser
+    KUZU_ROOT=str(KUZU_ROOT).replace("\\", "/")
+
+KUZU_EXEC_PATH = os.path.join(KUZU_ROOT, 'build', 'release', 'tools', 'shell', 'kuzu_shell')
+
+def _get_kuzu_version():
```

Comment:
Why do we add a _ before the function name?

Reply:
Python naming convention to mark a function as a private function